app-arch/smpq/smpq-1.6.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="StormLib MPQ archiving utility"
+DESCRIPTION="SMPQ is StormLib MPQ archiving utility. This utility is designed for full \
+manipulation of Blizzard MPQ archives."
+HOMEPAGE="https://launchpad.net/smpq"
+COPYRIGHT="2010-2016 Pali"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://launchpad.net/smpq/trunk/$portVersion/+download/smpq_$portVersion.orig.tar.gz"
+CHECKSUM_SHA256="b5d2dc8a5de8629b71ee5d3612b6e84d88418b86c5cd39ba315e9eb0462f18cb"
+SOURCE_DIR="smpq-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	smpq$secondaryArchSuffix = $portVersion
+	cmd:smpq$secondaryArchSuffix = $portVersion
```

Comment:
You use `$prefix/bin` below (instead of `$binDir`), so this shouldn't have a suffix.

```suggestion
	cmd:smpq = $portVersion
```